tests/config.json

```diff
@@ -0,0 +1,32 @@
+{
+    "details": {
+        "server_url": "http://138.217.142.94:8108",
+        "private_key": "0000000000000000000000000000000000000000000000000000000000000000",
+        "chain_id": 10143,
+        "pedantic_checks": false
+    },
+    "verify": {
+        "provider_url": "https://data-seed-prebsc-1-s3.binance.org:8545",
```

Comment:
As this repo is for public usage, so better here we should not use detail , should input <network-rpc>, <task-id> inside the ""